dvc/repo/plots/template.py

```diff
@@ -107,13 +107,15 @@ def _check_field_exists(data, field):
             raise NoFieldInDataError(field)
 
 
-class DefaultLinearTemplate(Template):
+class DefaultTemplate(Template):
     DEFAULT_NAME = "default"
 
     DEFAULT_CONTENT = {
         "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
         "data": {"values": Template.anchor("data")},
         "title": Template.anchor("title"),
+        "width": 300,
+        "height": 300,
```

Comment:
Since we do not store those values anymore in json files, they are not straightforward to edit, without creating a custom template. It seems to me that a little bigger plots are desired.